risc0/zkvm/src/prove/mod.rs

```diff
@@ -193,6 +193,9 @@ cfg_if::cfg_if! {
         use risc0_circuit_rv32im::cuda::CudaEvalCheck;
         use risc0_zkp::hal::cuda::CudaHal;
 
+        /// Returns the default SHA-256 HAL for the RISC Zero circuit
+        ///
+        /// See the non-cuda version of this documentation for details
```

Comment:
Does this show up in the docs? I don't think it will actually be visible unless these features are set.

Reply:
It does not, but if it's missing the `cuda` feature can't build due to `deny(missing_docs)` (see [this build](https://github.com/risc0/risc0/actions/runs/4347444670/jobs/7594765734)). Hence my "See the non-cuda version of this documentation for details" instead of a full duplication of the documentation.

Happy to hear other approaches for addressing this failure.

Reply:
I'm just thinking we can drop the reference to the non-cuda version